src/Elastic.Transport/Components/Providers/RecyclableMemoryStream.cs

```diff
@@ -283,17 +283,6 @@ protected override void Dispose(bool disposing)
 
 			RecyclableMemoryStreamManager.EventsWriter.MemoryStreamFinalized(_id, _tag, AllocationStack);
 
-#if !NETSTANDARD1_4
```

Comment:
Should this one not stay? But remove the ifdef because it's always NOT Net standard 1.4?